terraform/Yandex Cloud/main.tf

```diff
@@ -42,6 +42,7 @@ resource "yandex_compute_instance" "msc_time-vm" {
   }
 
   metadata = {
-    ssh-keys = "ubuntu:${file("~/.ssh/id_rsa.pub")}"
+    #ssh-keys = "ubuntu:${file("~/.ssh/id_rsa.pub")}"
+    ssh-keys = "ubuntu:${file("//wsl$/Ubuntu-20.04/home/andrew/.ssh/id_rsa.pub")}"
```

Comment:
oh come on, just use `~/.ssh/id_ed25519.pub` instead of full path ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)

the `~` thing even works on stupid os windows :trollface: 
well, it won't work if you mix both windows and wsl cos `~` will point to different locations but you get the idea ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)